modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,22 @@
+// Module filename: nw-customize-ingress-error-pages.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-customize-ingress-error-pages_{context}"]
+= Customizing Ingress Controller error pages
+
+As a cluster administrator, you can use the `HttpErrorCodePages` field to specify custom error code pages for the Ingress Controller. When you specify a custom error code page, your specified HTTP 404 error page is returned when the requested URI does not exist, instead of returning the HTTP 503 error page.
+
+.Procedure
+. Create a configmap named "my-custom-error-code-pages" in the openshift-config namespace.
+
+. Patch the ingresscontroller to reference the "my-custom-error-code-pages" configmap by name.
++
+[source,terminal]
+----
+$ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"HttpErrorCodePages": "my-custom-error-code-pages"}}' --type=merge
+----
++
+The "my-custom-error-code-pages" configmap is synched from the openshift-config namespace to the openshift-ingress namespace.
+
+Then, the ingress operator configures the router deployment with a volume and volume mount that mounts the custom error pages from the http503page configmap on top of the existing error pages in the router pods.
```

Comment:
Then, the ingress operator configures the router deployment with a volume.  Configmap with the custom error pages is mounted on the router volume on top of the existing error pages in the router pods.